tests/samplers_tests/test_samplers.py

```diff
@@ -775,6 +808,8 @@ def test_sample_single_distribution(sampler_class: Callable[[], BaseSampler]) ->
         "d": IntUniformDistribution(low=1, high=1),
         "e": IntLogUniformDistribution(low=1, high=1),
         "f": CategoricalDistribution([1]),
+        "j": IntDistribution(low=1, high=1),
```

Comment:
[memo] `g` `h` `i` will be used in https://github.com/optuna/optuna/pull/3111/files#diff-8b85ccf8a3534b2aa878bce8ece8ce930af410ab178ebc6d80031badbabb5052R812-R814.

Reply:
That's great! I was worried we would have conflicts between those two PRs, so that's a really nice touch.